### ISSUE TITLE
Break since 2016-11.4 (Peppet --client=runner manage.status)

### ISSUE THREAD

**Issue by PierreR** [saltstack/pepper#102]
After upgrading the salt master from `2016-11.3` to  `2016-11.4`, the command `peppet --client=runner manage.status` breaks with the following error:
```
export SALTAPI_USER="john"
export SALTAPI_PASS="secret"
export SALTAPI_URL="${salt-url}"
export SALTAPI_EAUTH=ldap

pepper --client=runner manage.status
{
    "return": [
        "Exception occurred in runner manage.status: Traceback (most recent call last):\n  File \"/usr/lib/python2.7/site-packages/salt/client/mixins.py\", line 395, in _low\n    data['return'] = self.functions[fun](*args, **kwargs)\nTypeError: status() got an unexpected keyword argument '__current_eauth_groups'\n"
    ]
}
```
On the master `salt-run manage.status --eauth=ldap --username=john --password=secret` does work.

I am also using `pepper --client=runner doc.execution` which is failing with
```
{
    "return": [
        "Exception occurred in runner doc.execution: Traceback (most recent call last):\n  File \"/usr/lib/python2.7/site-packages/salt/client/mixins.py\", line 395, in _low\n    data['return'] = self.functions[fun](*args, **kwargs)\nTypeError: execution() takes no arguments (2 given)\n"
    ]
}
```

The problem might be related to `salt-api`. Will have to rollback to `2016-11.3`.

**Comment by PierreR** [comment]
Might be related to https://github.com/saltstack/salt/issues/40845

**Comment by PierreR** [comment]
Fix in 2016.11.5
